openapi/components/schemas/Invoices/InvoiceItem.yaml

```
@@ -39,6 +39,20 @@ properties:
     description: ID of the product.
     maxLength: 50
     example: prod_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  planId:
+    type: string
+    nullable: true
+    description: ID of the plan.
+    maxLength: 50
+    readOnly: true
+    example: plan_0YV7DES3WPC5J8JD8QTVNZBZNZ
```

I think it will be better to use the same ULID per resource for consistency

```suggestion
    example: plan_0YV7DENSVGDBW9S71XZNNYYQ0X
```